desktop/package.json

```diff
@@ -51,6 +50,7 @@
 		"webpack-cli": "^4.9.2"
 	},
 	"dependencies": {
+		"@electron/rebuild": "3.2.13",
```

Comment:
`electron-rebuild` should not be a dev dependency, and not a direct dependency of the app.